client/src/components/Login/Login.js

```diff
@@ -107,38 +108,35 @@ export class Login extends Component {
 			network: {
 				error: null,
 				value: networks[0].name || ''
-			}
+			},
+			authEnabled: networks[0].authEnabled
 		}));
 	}
 
 	handleChange = event => {
 		const { target } = event;
 		const value = target.type === 'checkbox' ? target.checked : target.value;
 		const { name } = target;
-		this.setState({
-			[name]: { value }
-		});
-	};
 
-	submitForm = async e => {
-		e.preventDefault();
-
-		const { user, password, network } = this.state;
+		const newState = {
+			[name]: { value }
+		};
+		if (name === 'network') {
+			const { networks } = this.state;
+			newState.authEnabled = (networks.find(n => n.name === value) || {}).authEnabled;
+		}
 
-		await this.performLogin({
-			user: user.value,
-			password: password.value,
-			network: network.value
-		});
+		this.setState(newState);
 	};
 
 	async performLogin({ user, password, network}) {
 		const { login } = this.props;
+		const { authEnabled } = this.state;
 
 		const info = await login(
 			{
-				user,
-				password,
+				user: authEnabled ? user : 'dummy-user',
```

Comment:
can we use `user` and `password` instead?

Reply:
I prefer to be explicit here, just `'user'` seemed a bit weird